tests/letstest/targets.yaml

```diff
@@ -13,40 +24,25 @@ targets:
     type: ubuntu
     virt: hvm
     user: ubuntu
-  - ami: ami-09677e0a6b14905b0
-    name: ubuntu16.04LTS
-    type: ubuntu
-    virt: hvm
-    user: ubuntu
   #-----------------------------------------------------------------------------
   # Debian
   - ami: ami-01db78123b2b99496
     name: debian10
     type: ubuntu
     virt: hvm
     user: admin
-  - ami: ami-003f19e0e687de1cd
-    name: debian9
-    type: ubuntu
-    virt: hvm
-    user: admin
-  - ami: ami-0ed54dd1b25657636
-    name: debian9_arm64
-    type: ubuntu
-    virt: hvm
-    user: admin
-    machine_type: a1.medium
-    # userdata: |
-    #   #cloud-init
-    #   runcmd:
-    #     - [ apt-get, install, -y, curl ]
   #-----------------------------------------------------------------------------
   # Other Redhat Distros
   - ami: ami-0916c408cb02e310b
     name: RHEL7
     type: centos
     virt: hvm
     user: ec2-user
+    userdata: |
+      #cloud-config
+      runcmd:
+        - yum-config-manager --enable rhui-REGION-rhel-server-extras rhui-REGION-rhel-server-optional
```

Comment:
Do you know if there's a potential race condition where our test farm test scripts start before this command is executed?

Reply:
we rely on `runcmd` elsewhere so I had assumed not, but that could explain the one spurious failure in azure that I obviously couldn't check the logs on.

Reply:
I'm not sure, but I suspect the other `runcmd` commands aren't really needed.

I personally think we should either convince ourselves we're not adding a race condition here or move this code to bootstrap_os_packages.sh. I think the latter shouldn't be too hard. I'm imagining something like:
```
BootstrapRpmPython3() {
  InitializeRPMCommonBase

  python_pkgs="python3
    python3-devel
  "

  if $TOOL list python3-devel >/dev/null 2>&1; then
    yum-config-manager --enable rhui-REGION-rhel-server-extras rhui-REGION-rhel-server-optional
    python_pkgs="$python_pkgs
      python3-devel.x86_64
    "
  fi

 BootstrapRpmCommonBase "$python_pkgs"
}
```
but I didn't test that code.

Reply:
I'm fine with just moving it there.